src/lightkurve/io/cdips.py

```diff
@@ -74,8 +85,19 @@ def read_cdips_lightcurve(filename,
     quality_mask = (lc['quality']=="G") | (lc['quality']=="0")
     lc = lc[quality_mask]
 
+
+    #This makes sure that the TIC ID is obtained and returned as the plot label
+    tic = lc.meta.get('TICID')
+    if tic is not None:
+        # compatibility with SPOC, QLP, etc.
+        lc.meta["TARGETID"] = tic
+        lc.meta["TICID"] = tic
+        lc.meta["OBJECT"] = f"TIC {tic}"
+        # for Lightkurve's plotting methods
+        lc.meta["LABEL"] = f"TIC {tic}"
+
+    
     lc.meta["AUTHOR"] = "CDIPS"
-    lc.meta['TARGETID'] = lc.meta.get('TICID')
     lc.meta['QUALITY_BITMASK'] = 36
```

Comment:
This needs to be updated

Reply:
Is the point here to change the quality bitmask to the TESS default i.e., 175?

Reply:
Additionally CDIPS light curves have already had quality filtering applied, and do not provide the bitflags necessary for a user to apply a new bitmask.